runtime/src/account_info.rs

```diff
@@ -87,20 +87,30 @@ pub struct AccountInfo {
     /// index identifying the append storage
     store_id: AppendVecId,
 
+    account_offset_and_flags: AccountOffsetAndFlags,
+}
+
+#[derive(Default, Debug, PartialEq, Eq, Clone, Copy)]
+pub struct AccountOffsetAndFlags {
```

Comment:
this will be the type passed as what we save on disk in the disk index.

Reply:
I was imagining the bitfield `PackedOffsetAndFlags` was low level and I wanted to make it not public. Looks like it is public. I may work on that. I wanted a type that was more clear what it did and that it could be serialized.